lib/src/node/node.dart

```diff
@@ -49,8 +52,9 @@ class Node extends HiveObject {
       final url = Uri.http(uri, '/json_rpc');
       final headers = {'Content-type': 'application/json'};
       final body = json.encode(requestBody);
+      print('Node Port: ${url.port}');
```

Comment:
Leftover debugging?

Reply:
No actually that is the Fix. 
For some reason I have to evaluate the port of the Url before the request. That is done here.